services/AUTOMATIC1111/Dockerfile

```diff
@@ -78,6 +79,7 @@ RUN <<EOF
 chmod +x /docker/mount.sh && python3 /docker/info.py ${ROOT}/modules/ui.py
 EOF
 
+RUN mkdir ${ROOT}/interrogate && cp ${ROOT}/repositories/clip-interrogator/data/* ${ROOT}/interrogate
```

Comment:
move to below `COPY --from=download /git/ ${ROOT}` so the copy does not happen on every update of UI